src/tactCompiler.ts

```diff
@@ -39,14 +39,23 @@ export class TactCompiler {
         }
 
         const pathKey = path.relative( this.rootPath, args.file).replaceAll('\\','/');
+
+        let rootPath = this.rootPath;
```

Comment:
Could we change `let rootPath = this.rootPath;` on `let localRootPath = this.rootPath;` ? just because we can misunderstand later `this.rootPath` and `rootPath`
The same below by code. We can leave this, of course, just to better read the code.
Please decide.

Reply:
Yes that would be a better.